unit_tests/test_views.py

```diff
@@ -27,9 +27,11 @@ def test_index(self):
         self.assertContains(response, 'The Theater that was Rome')
 
     def test_about(self):
+        models.Static.objects.create(title='About', text='### Red Sox lineup[^n1]\n\n[^n1]: footnote text')
         response = self.client.get(reverse('about'))
         self.assertEqual(response.status_code, 200)
-        self.assertContains(response, '<h2>About</h2>')
+        self.assertContains(response, 'Red Sox lineup') #make sure that basic markdown was rendered
```

Comment:
Sorry, let me try again. The assertContains text should be
`'<h3>Red Sox lineup'`
(with the h3 element).